client/lib/abtest/active-tests.js

```diff
@@ -107,4 +107,14 @@ module.exports = {
 		},
 		defaultVariation: 'original'
 	},
+	promoteFreeDomain: {
+		datestamp: '20160301',
+		variations: {
+			original: 50,
+			freeDomain: 50
+		},
+		defaultVariation: 'original',
+		excludeSitesWithPaidPlan: true
```

Comment:
I don't think we really need this, since only new users will be included, but it doesn't hurt to put it in.